misc/sampleapp-android-tests/sampleapp/spacedodgegame_base_appFunction.html

```diff
@@ -27,13 +27,14 @@
 
 Authors:
     Wang, Hongjuan<hongjuanx.wang@intel.com>
+    Li, Hao<haox.li@intel.com>
```

Comment:
Shall this follow the guide "Put test author into metadata" at https://github.com/crosswalk-project/crosswalk-test-suite/commit/25832c28c43b5c44944cc0861a4a281f5fed2da0?